Documentation/packaging.md

```diff
@@ -2,32 +2,9 @@
 
 This document aims to provide information about packaging rkt in Linux distributions. It covers dependencies, file ownership and permissions, and tips to observe packaging policies.
 
-## Build-time dependencies
+## Build-time and run-time dependencies
 
-### Basic build-time dependencies
-
-- autoconf
-- GNU Make
-- glibc-static
-- bash
-- go
-- gofmt
-- file
-- git
-- gpg
-
-#### Additional build-time dependencies for the coreos flavor
-
-wget, gpg, mktemp, md5sum, cpio, gzip, unsquashfs, sort
-
-#### Additional build-time dependencies for the kvm flavor
-
-patch, bc
-
-#### Additional build-time dependencies for the src flavor
-
-intltoolize, libtoolize and all systemd dependencies.
-The dependencies may differ depending on the version of built systemd.
+Please see [the list of dependencies][deps].
```

Comment:
With this change, should the next section "Offline builds" still be inside the "Build-time and run-time dependencies" section?